pyproject.toml

```diff
@@ -8,10 +8,10 @@ authors = ["Dan Bentley <dan@numiko.com>"]
 python = "^3.8"
 gunicorn = "^20.1.0"
 elasticsearch = "^7.12.0"
-wagtail = "^2.12.3"
+wagtail = "2.15"
```

Comment:
2.15.1 is now released, which contains some bug fixes. Probably worth switching to that - it should be very straightforward